ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -11,9 +11,10 @@ workflow:
       ROSA_LOGIN_ENV: "integration"
     pre:
     - ref: osd-ccs-fleet-manager-sc-provision
-    - ref: osd-ccs-fleet-manager-test
     - chain: rosa-aws-sts-provision-byo-vpc
     - ref: osd-ccs-conf-idp-htpasswd-multi-users
+    test:
+    - ref: osd-ccs-fleet-manager-test
```

Comment:
I was thinking the test ref shall be in ci job 'aws-ocm-osd-ccs-fleet-manager-f7' instead of workflow.
Not sure if it is ok/appropriate to put it here.
See https://steps.ci.openshift.org/workflow/generic-claim
@yasun1 @LiangquanLi930 Do you think that? Thx

Reply:
+1